src/main/java/org/jenkinsci/plugins/workflow/actions/PersistentAction.java

```diff
@@ -0,0 +1,10 @@
+package org.jenkinsci.plugins.workflow.actions;
+
+import hudson.model.Action;
+
+/**
+ * Marker interface for an action that can't be contributed by a {@link jenkins.model.TransientActionFactory}
+ * Actions implementing this can use more efficient getAction methods
```

Comment:
Either `@link` to a specific method, or replace with a generic promise that something more efficient is possible.